src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
@mikem8361 according to @jkotas, the only PAL that should be relying on `SIGRTMIN` is the copy linked into libcoreclr, where that signal is used for thread suspension. Are you referring to `SIGTERM`? We don't chain that signal now (at least not once the PAL has been initialized), but I agree that we should.